apps/openmw/mwphysics/heightfield.hpp

```diff
@@ -27,6 +32,9 @@ namespace MWPhysics
         btHeightfieldTerrainShape* mShape;
         btCollisionObject* mCollisionObject;
         osg::ref_ptr<const osg::Object> mHoldObject;
+#ifdef BT_USE_DOUBLE_PRECISION
+        std::vector<btScalar> mHeights;
```

Comment:
BTW, is it possible to use the `mHeights` in both precisions to avoid conditional compilation?

Reply:
Yes, it's possible.

Reply:
Are there drawbacks here? Maybe it would be better indeed to use the BT_USE_DOUBLE_PRECISION only for unit tests.

Reply:
Unnecessary copying of heights data for single precision.

Reply:
Or we can use SFINAE to avoid copying. But not sure is it better than conditional compilation here.